gateway-js/src/executeQueryPlan.ts

```diff
@@ -326,12 +327,26 @@ async function executeNode(
       }
       return new Trace.QueryPlanNode({ fetch: traceNode });
     }
+    case 'Condition': {
+      // TODO: we probably should a add a proper protobuf message for this in https://github.com/apollographql/apollo-server/blob/main/packages/usage-reporting-protobuf/src/reports.proto
```

Comment:
This is fixed in the latest version of the protobuf package (just published today)

Reply:
Nice. Added.